integration/container/run_test.go

```diff
@@ -0,0 +1,44 @@
+package container // import "github.com/docker/docker/integration/container"
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/versions"
+	"github.com/docker/docker/integration/internal/container"
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/poll"
+	"gotest.tools/v3/skip"
+)
+
+func TestConsoleSize(t *testing.T) {
+	skip.If(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.42"), "skip test from new feature")
+
+	defer setupTest(t)()
+	client := testEnv.APIClient()
+	ctx := context.Background()
+
+	cID := container.Run(ctx, t, client,
+		container.WithTty(true),
+		container.WithImage("debian:bullseye-slim"),
```

Comment:
Was wondering why not busybox, but looks like that doesn't support `tput` 🤷‍♂️

Reply:
Yeah, unfortunately it doesn't. However some other tests also use this image, so I hope that it doesn't actually make the test suite heavier :)

Reply:
It's part of the "frozen images" which are only pulled once, and loaded into test-daemons, so it should not make a huge difference https://github.com/moby/moby/blob/c9e19a2aa182e06ebe1653a2c3af02563acf9e36/Dockerfile#L88-L93


We should look at only conditionally loading some of those though, as I think we currently always load all of them in every test daemon

Reply:
I think it's possible to use busybox by using `stty size` instead of tput:
`docker run -it busybox sh -c 'stty size'`
which returns height and width like this (it even follows our favorite order! :smile:):
`51 234`

The only caveat is that with the previous behaviour (no console size being set on creation) stty does not output the size, but:
`stty: standard input`